### PR TITLE
Updated issue templates from .github.

### PR DESCRIPTION
**Pull request by dblock**
Signed-off-by: dblock <dblock@dblock.org>

### Description

Coming from https://github.com/opensearch-project/.github/issues/64, updated issue templates for bugs and features.
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peternied — APPROVED** [review]
Thanks for taking this on

**Review by peternied — DISMISSED** [review]

**Review by peternied — APPROVED** [review]

**Comment by dblock** [comment]
@peternied Looks like CI is failing with real albeit unexpected failures, help?

**Comment by peternied** [comment]
Looks like main is broken due do the default version of dashboards changing out from underneath this repo.  I'll make sure another maintainer approves this change and then I'll merge it past the build break.  Tracking issue, https://github.com/opensearch-project/security-dashboards-plugin/issues/932

Not-so-useful failure:
> Plugin "securityDashboards" is only compatible with OpenSearch Dashboards version "1.3.0", but used OpenSearch Dashboards version is "1.4.0". (incompatible-version, /home/runner/work/security-dashboards-plugin/security-dashboards-plugin/OpenSearch-Dashboards/plugins/security-dashboards-plugin/opensearch_dashboards.json)

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+contact_links:
+  - name: OpenSearch Community Support
+    url: https://discuss.opendistrocommunity.dev/
+    about: Please ask and answer questions here.
+  - name: AWS/Amazon Security
+    url: https://aws.amazon.com/security/vulnerability-reporting/
+    about: Please report security vulnerabilities here.
```
Comment (cliu123) [inline .github/ISSUE_TEMPLATE/config.yml:7]: PR looks good to me! Please add EOF.

**Review by cliu123 — APPROVED** [review]

**Comment by cliu123** [comment]
Please squash when merging. : )

**Review by peternied — APPROVED** [review]

**Pull request merged**